packages/local_auth/android/src/main/java/io/flutter/plugins/localauth/AuthenticationHelper.java

```diff
@@ -136,12 +143,13 @@ public void onAuthenticationError(int errorCode, CharSequence errString) {
 
   @Override
   public void onAuthenticationSucceeded(BiometricPrompt.AuthenticationResult result) {
+    Log.d("LocalAuth", "Authentication Succeed");
     stop(true);
   }
 
   @Override
   public void onAuthenticationFailed() {
```

Comment:
That means the failed() callback really is a noop from the plugin perspective. What happens when maximum number of authentication attempts is reached?

Reply:
When maximum attempt is reached, i guess onAuthenticationError will be triggered.

Reply:
onAuthenticationFailed callback is only triggered when biometric is not recognized, which is automatically maintained by the biometric prompt.

Reply:
@mehmetf , added error codes **LockedOut** and **PermanentlyLockedOut** to detect maximum number of authentication attempts being reached.